lua/gatekeeper/staticlib.lua

```diff
@@ -19,6 +19,14 @@ function get_numa_table (net_conf)
 	return numa_table
 end
 
+function count_numa_nodes (numa_table)
```

Comment:
Move the code of `count_numa_nodes()` just before `alloc_lcores_evenly_from_all_numa_nodes()`. There's no reason to spread `count_numa_nodes()` from its first caller.